Task3/Task3_Atanas_Kisyov/README.md

```diff
@@ -0,0 +1,13 @@
+# Connect to PostgreSQL with Python
+
+## View Table
+
+```shell
+python3 src/python/main.py r
```

Comment:
When i execute a script, script should create read pool or write pool. I propose to use `-ro or --ro-access`. The same with write.